app/grandchallenge/products/models.py

```diff
@@ -113,6 +113,7 @@ class FDAStatus(models.TextChoices):
     ce_status = models.CharField(
         choices=CEStatus.choices, max_length=3, default=CEStatus.NO
     )
+    ce_under = models.CharField(max_length=10, default="")
```

Comment:
You could also use `blank=True` instead of `default=""`. That is a bit more common.